File: semana03/flask-ecommerce/app/schemas/auth_schema.py
from pydantic import BaseModel

class RegisterSchema(BaseModel):
    name: str
    last_name: str
    email: str
    password: str
    role_id: int

class UserSchema(BaseModel):
    id: int
    name: str
    last_name: str
    email: str
    status: bool
    created_at: str
    updated_at: str
    role_id: int

File: semana09/express_chat_app/src/schemas/auth.schema.ts
import { z } from "zod";

export const registerSchema = z.object({
  username: z.string().trim(),
  email: z.string().email().trim(),
  password: z.string().trim(),
});

export const loginSchema = z.object({
  email: z.string().email().trim(),
  password: z.string().trim()
})